src/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/PhoneNumberInput.tsx

```diff
@@ -74,10 +76,13 @@ export const PhoneNumberInput: React.FC<PhoneNumberInputProps> = ({
 
   return (
     <Flex flexDirection="column" {...props}>
-      <Flex justifyContent="space-between">
-        <Text variant="xs" mb={0.5}>
+      <Flex>
+        <Text variant="xs" mb={0.5} mr={0.5}>
           Phone number
         </Text>
+        <Text variant="xs" color="black60">
+          (Optional)
+        </Text>
```

Comment:
**Issue:** This will be always displayed, even if it's not optional

Reply:
Thanks! 🙇  Fixed this part as well.